src/include/main/db_main.h

```diff
@@ -393,6 +427,16 @@ class DBMain {
                                            common::ManagedPointer(log_manager), create_default_database_);
       }
 
+      std::unique_ptr<storage::RecoveryManager> recovery_manager = DISABLED;
```

Comment:
Just for my own clarification, is replication the only current use for the `RecoveryManager`? Is there any other way to enable it?

Reply:
I believe `recovery_test` checks that it actually works, but otherwise, yes. `RecoveryManager` prior to this PR is dead code afaik.